_pages/blogs/GSoD23_QuillPusher.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Recap of Compiler Research Organization’s participation in Google’s Season of Docs 2023"
+layout: gridlay
+excerpt: "A review of the documentation efforts in 2023."
+sitemap: false
+permalink: blogs/gsod23_quillpusher_experience_blog/
+---
+
+
+# Recap of Compiler Research Organization’s participation in Google’s Season of Docs 2023
+
+## How we got started
+
+An audit of the existing documentation was done  to identify the gaps and
+potential areas of improvement. Our main areas of focus were (these evolved
+over time, based on documentation audit):
+
+- Automatic Differentiation applications (using Clad, e.g. in RooFit and
+  Floating-Point Error Estimation), and
+
+- Python-C++ Interoperability (Clang-Repl (LLVM), CppInterOp, cppyy, Numba,
+  etc.).
+
+We also wanted to capture the [Student Success Stories], including milestones
+achieved by student developers under the banner of Compiler Research
+Organization’s guidance and mentorship programs.
+
+## What we accomplished
+
+Following are the major documentation areas that we worked on.
+
+- AD Support in RooFit (ROOT)
+- Execution Results Handling in Clang-REPL (LLVM)
+- Floating Point Error Estimation (Clad)
+- CppInterOp for Python and C++ Interoperability (Compiler Research
+  Organization)
+- cppyy Enhancements (Compiler Research Organization)
+- Numba Enhancements (cppyy)
```

Comment:
It might be a good idea to make these links to the projects, blog, tutorials, etc.

Reply:
done